lib/routes.dart

```diff
@@ -39,6 +43,10 @@ class Routes {
       '/family-members': (BuildContext ctx) => const FamilyMembersScreen(),
       '/mainscreen': (BuildContext ctx) => const BottomBar(title: 'PICOS',),
       '/studynursescreen': (BuildContext ctx) => const StudyNurseScreen(),
+      '/configuration-screen': (BuildContext ctx) => const ConfigurationScreen(),
+      '/configuration-pages': (BuildContext ctx) => const ConfigurationPages(),
+      '/configuration-finish-screen': (BuildContext ctx) => const ConfigurationFinishedScreen(),
+      '/overview': (BuildContext ctx) => const OverviewScreen(),
     };
   }
```

Comment:
Please keep in mind that we shouldn't use a flat route structure, please let's have a tree like structure and introduce route names like `/studynursescreen/configuration-page`

Reply:
I've implemented your suggestion. You may have a look :)